### PR TITLE
Update Project Portfolio Page - Sagar

### PR DESCRIPTION
**Pull request by sagarsureka**

### CODE REVIEW

File: docs/team/sagarsureka.md [hunk 1]
```diff
@@ -3,44 +3,64 @@ layout: page
...
+  * Improved the responses given to the user by making it more readable and informative. [\#79](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/79)  
+  * Improved the product User Guide by fixing bugs reported in the Mock Practical Exam. [\#174](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/174)  
+
+* **Project Management and contribution to team-based tasks**:
+  * Participated in the planning of the project idea (Management of food inventory to reduce food wastage).
+  * Participated in drafting the initial requirements for SimplyKitchen - *Product Scope, Glossary, User Stories, Use Cases and Non-Functional Requirements.* 
+    This can be found in our [Developer Guide requirements appendix](https://ay2021s1-cs2103t-f13-4.github.io/tp/DeveloperGuide.html#appendix-requirements).
+  * Compiled the comments received on the team's User and Developer Guides from peers.
+  * Responsible for proper integration of the various parts of the software to form a whole.
+  * Responsible to ensure that the project code maintains high coding standards.
```
Comment (bchenghi) [inline docs/team/sagarsureka.md]: Perhaps this could be "Responsible for ensuring"?

File: docs/team/sagarsureka.md [hunk 2]
```diff
@@ -3,44 +3,64 @@ layout: page
 title: Sagar Sureka's Project Portfolio Page
 ---
```
Comment (bchenghi) [inline docs/team/sagarsureka.md:5]: Perhaps we can remove the above 5 lines?

File: docs/team/sagarsureka.md [hunk 3]
```diff
@@ -3,44 +3,64 @@ layout: page
...
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+* **Code contributed**: [Link to RepoSense](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=Sagar&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=sagarsureka&tabRepo=AY2021S1-CS2103T-F13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code)
 
+* **Enhancements**:
+  * Added a **quantity field** for food items:
+    * What it does: <br>Allows the user to track the quantity of each food item.
+    * Justification: <br>The features improves the application significantly as it helps manage another entity relevant to food items.
+    * Highlights: <br>This enhancement involved consideration of multiple alternatives of implementation. 
+    Keeping to the criteria of making it user friendly, the `unit` of quantity field was made optional and customisable. 
+    Instead of giving some specfic options, the user can have any word as the unit.
```
Comment (bchenghi) [inline docs/team/sagarsureka.md]: There's a typo for "specific".

**Review by bchenghi — COMMENTED** [review]
It looks good! I added some comments.

File: docs/team/sagarsureka.md (hunk 3 above)
Comment (sagarsureka) [reply docs/team/sagarsureka.md]: Thanks!

File: docs/team/sagarsureka.md (hunk 1 above)
Comment (sagarsureka) [reply docs/team/sagarsureka.md]: Thanks!

**Review by bchenghi — COMMENTED** [review]
LGTM

**Pull request merged**